src/UserSpaceInstrumentation/InstrumentProcess.cpp

```diff
@@ -262,8 +262,10 @@ ErrorMessageOr<absl::flat_hash_set<uint64_t>> InstrumentedProcess::InstrumentFun
                              entry_payload_function_address_, return_trampoline_address_,
                              capstone_handle, relocation_map_);
         if (address_after_prologue_or_error.has_error()) {
-          ERROR("Failed to create trampoline: %s",
-                address_after_prologue_or_error.error().message());
+          const std::string message = absl::StrFormat(
+              "Failed to create trampoline: %s", address_after_prologue_or_error.error().message());
```

Comment:
How is this error associated with the corresponding function? Also see the other comment.